frontend/packages/policy-editor/src/components/PolicyCardRules/PolicyRule/PolicyAccessPackages/PolicyAccessPackageAccordion/PolicyAccessPackageAccordion.module.css

```diff
@@ -0,0 +1,50 @@
+:root {
+  --logo-size: 20px;
+}
+
+.accessPackageAccordion {
+  background-color: var(--brand-default-third-100);
+}
+
+.serviceContainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: var(--fds-spacing-2);
+  background-color: var(--fds-semantic-background-default);
+  padding: 0 var(--fds-spacing-2);
+  min-height: var(--fds-spacing-8);
+  font-size: var(--fds-sizing-3);
+  box-shadow: var(--fds-shadow-small);
+}
+
+.serviceLabel {
+  flex: 1;
+  font-weight: 500;
+}
+
+.logo {
+  max-height: var(--logo-size);
+  max-width: var(--logo-size);
+}
+
+.emptyLogo {
+  height: var(--logo-size);
+  width: var(--logo-size);
+}
+
+.serviceContainerHeader {
+  color: var(--colors-grey-700);
+  font-size: var(--fds-sizing-3);
+  padding-top: var(--fds-spacing-2);
+  border-top: 1px solid var(--fds-semantic-border-neutral-subtle);
+}
+
+.noServicesText {
+  font-size: var(--fds-sizing-4);
```

Comment:
What is the reason that we set the font-size for texts instead of using the `size` prop? 😄 
`size='sm'` or `size='xs'`